### ISSUE TITLE
Improvement: Cloudflare deploy production explanation

### ISSUE THREAD

**Issue by Thomvh** [celzero/docs#7]
Currently the default cloudflare deployment deploys everything as development with the environment variables editable using the worker UI and never an update.

**Comment by ignoramous** [comment]
Docs are super out of date. Looking to fix docs over the coming weeks.

**Comment by hgareeballa** [comment]
Is there any way to secure the configuration screen ! as at the moment any one could change my dns configuration ?

**Comment by ignoramous** [comment]
> Is there any way to secure the configuration screen ! as at the moment any one could change my dns configuration?

No, they can't. The configuration (which looks like: `https://max.rethinkdns.com/1:4P___________________________-D_`) is already stored in the URL (specifically, the part that goes `1:4P___________________________-D_` is the configuration).

**Comment by hgareeballa** [comment]
i have deployed a serverless dns to cloudflare and i can access the /configuration url from any wr !

**Comment by ignoramous** [comment]
> i have deployed a serverless dns to cloudflare and i can access the /configuration url from any wr !

You mean, the domain is publicly accessible? You can setup auth, but typically, you don't have to worry about anyone hitting your URL endpoint if you don't advertise it anywhere. Workers today costs $1 per every 3 million to 5 million requests (that's worth ~20 devices worth of DNS queries).

If you still want auth, check the [serverless-dns readme](https://github.com/serverless-dns/serverless-dns/?tab=readme-ov-file#auth)
